proto/wtools/abase/l0/l5/Str.s

```diff
@@ -2553,6 +2553,18 @@ function strSplitInlinedStereo_( o )
   _.assert( arguments.length === 1, 'Expects single argument' );
```

Comment:
save original version of the routine in comments, please.
make sure it does not brake code where strSplitInlinedStereo_ is used.